functions/defaulted-parameters.md

```diff
@@ -53,18 +53,18 @@ A student is able to:
 
 A student should be able to:
 
-1. Call to a function with a default parameter with or without that parameter specified
-2. Declare a function with a default parameter, and omit the default in the definition's signature
-3. Explain what the requirements are for a type to be usable with a default parameter
-4. Explain when the lifetime of a defaulted parameter begins and ends
+1. Call to a function with a default argument with or without that argument specified
+2. Declare a function with a default argument, and omit the default in the definition's signature
+3. Explain what the requirements are for a type to be usable as a default argument
```

Comment:
This doesn't make sense to me: a type is not used as a default argument, an initializer is.

Reply:
Would such be better?
"Explain what the requirements are on a type for a value of that type to be usable as a default argument"

Reply:
*Are* there any requirements on a type for a value of that type to be usable as a default argument? I can't think of any such requirements off the top of my head. You want a default argument of type `std::map<int, std::string>`?— no problem.

Reply:
What's the current status of this? Should we adapt the alternative wording from @rwdougla ?

Reply:
I've removed the bullet point. I can't even recall what I had in mind for it, anyway. If someone has an idea as to why it would be useful, we can always add it back in later